adafruit_bitmapsaver.py

```diff
@@ -106,7 +119,11 @@ def _write_pixels(output_file, pixel_source, palette):
 # pylint:enable=too-many-locals
 
 
-def save_pixels(file_or_filename, pixel_source=None, palette=None):
+def save_pixels(
+    file_or_filename: Union[str, BufferedWriter],
+    pixel_source: Union[Display, Bitmap] = None,
+    palette: Palette = None,
```

Comment:
This `Palette` should be optional I think since it's default value is None. 

I can add a commit with that change. It's looking ready to go after that to me.